src/Arcus.Scripting.Sql/Arcus.Scripting.Sql.psm1

```diff
@@ -1,3 +1,73 @@
+Class DatabaseVersion : System.IComparable
```

Comment:
Wondering if we should also make sure that this is formatted the same way as our other files 

```powershell
class DatabaseVersion : System.IComparable {
```

Reply:
You mean we should format it with the opening curly brace on the same line as the if statement or function/class declaration ?
I've modified the `DatabaseVersion` class now so that it is formatted like that.

Reply:
Then, the same should be done for the migration-script itself imho.

However, I can see that the formatting we're using is not consistent throughout the codebase.

Reply:
Changed the migrate-script as well to be conform with Powershell coding convention.